{"name": "获取同步根组id", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <bms.testcases.test_02_usersync.TestUserSync object at 0x0000025263D56D60>\ncaseid = 'usersync_get_syncgroupid_01'\ncasedata = {'base_url': '${config(host)}', 'caseid': 'usersync_get_syncgroupid_01', 'description': '获取同步根组id', 'epic': '用户管理', ...}\n\n    @pytest.mark.parametrize('caseid, casedata', [(caseid, casedata) for caseid, casedata in\n                                                  readcase.ReadCase().get_case_dict(\n                                                      ROOT_DIR + 'bms/data/usersync/usersync_get_syncgroupid.yaml').items()])\n    def test_usersync_get_syncgroupid(self, caseid, casedata):\n        handleallure.allure_display(casedata)\n        logging.info(\"==========开始执行用例：%s==========\", caseid)\n>       runcase.excute_case(casedata)\n\nbms\\testcases\\test_02_usersync.py:165: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ncommon\\runcase.py:54: in excute_case\n    checkresult.check_result(hope_result, recv_data, recv_code)\ncommon\\checkresult.py:30: in check_result\n    assert_response(value, real_res)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nhope_res = {'msg': 'success', 'statusCode': 0}\nreal_res = {'msg': '找不到对应的组织机构', 'statusCode': 1114886}\n\n    def assert_response(hope_res, real_res):\n        \"\"\"\n        返回结果校验 -全返回校验\n        :param hope_res:\n        :param real_res:\n        :return:\n        \"\"\"\n        try:\n            with allure.step(\"返回值校验\"):\n                allure.attach(name=\"期望返回值\", body=str(hope_res))\n                allure.attach(name='实际返回值', body=str(real_res))\n                if isinstance(hope_res, (dict, list)) and isinstance(real_res, (dict, list)):\n                    flag = handledict.cmp_dict(hope_res, real_res)\n>                   assert flag\nE                   AssertionError\n\ncommon\\checkresult.py:45: AssertionError"}, "description": "获取同步根组id", "steps": [{"name": "执行关联接口", "status": "passed", "steps": [{"name": "执行当前接口：同步任务列表搜索-无搜索条件", "status": "passed", "attachments": [{"name": "请求方法：", "source": "ff9811b2-9e74-4d31-858d-a51cb57c492b-attachment.attach"}, {"name": "请求地址", "source": "99d6cd02-c432-4015-ab65-7dacac0c17b6-attachment.attach"}, {"name": "请求头", "source": "3c7ba849-9fc6-4781-8dd4-a074030a1545-attachment.attach"}, {"name": "请求参数", "source": "e49acee7-deab-4c6c-80e7-c19df7500280-attachment.attach"}, {"name": "请求结果", "source": "997006e5-9a54-45e6-a9f2-013dd4b421c8-attachment.attach"}], "start": 1687760693827, "stop": 1687760693906}, {"name": "结果校验", "status": "passed", "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "5e60d03a-5a93-49f2-9f69-6c57e648fe70-attachment.attach"}, {"name": "实际状态码", "source": "7a31e93a-8c82-4cb6-af5a-8a511c0ab36b-attachment.attach"}], "start": 1687760693906, "stop": 1687760693907}, {"name": "返回值校验", "status": "passed", "attachments": [{"name": "期望返回值", "source": "a2b2193c-5c3b-4808-8a78-f420fca6d10a-attachment.attach"}, {"name": "实际返回值", "source": "9e3471f8-5ba5-42bd-b0c7-ec1bea3ff60e-attachment.attach"}], "start": 1687760693907, "stop": 1687760693911}], "start": 1687760693906, "stop": 1687760693911}], "start": 1687760693798, "stop": 1687760693911}, {"name": "执行当前接口：获取同步根组id", "status": "passed", "attachments": [{"name": "请求方法：", "source": "91ad047d-5df7-4400-a5bc-d2a2d19f9671-attachment.attach"}, {"name": "请求地址", "source": "0ba13103-0ce9-4af6-9dca-2be239804131-attachment.attach"}, {"name": "请求头", "source": "b681d788-5756-40f4-9740-deb32393b50e-attachment.attach"}, {"name": "请求参数", "source": "15bda7de-efcc-4639-a9bd-5ab0e6253ec0-attachment.attach"}, {"name": "请求结果", "source": "e22de0e4-1fa3-4790-9e6c-61cb3faad1eb-attachment.attach"}], "start": 1687760693915, "stop": 1687760693982}, {"name": "结果校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 30, in check_result\n    assert_response(value, real_res)\n  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "23dd8841-d405-4d71-9c97-4366af29f66f-attachment.attach"}, {"name": "实际状态码", "source": "736cd384-1720-4813-bc89-30bd74eadcb4-attachment.attach"}], "start": 1687760693982, "stop": 1687760693983}, {"name": "返回值校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 45, in assert_response\n    assert flag\n"}, "attachments": [{"name": "期望返回值", "source": "f80bce6b-3b61-42a4-870f-554d9f1de5e6-attachment.attach"}, {"name": "实际返回值", "source": "69e96dcf-6ab7-486d-afba-ca96cecbdb51-attachment.attach"}], "start": 1687760693983, "stop": 1687760693983}], "start": 1687760693982, "stop": 1687760693984}], "attachments": [{"name": "log", "source": "c34fe596-a1ba-4a8d-aef7-86f4c55906a4-attachment.txt", "type": "text/plain"}], "start": 1687760693796, "stop": 1687760693984, "uuid": "67302e01-89c1-45c1-8dcb-e2ead9f07a15", "historyId": "c076d0bb657b7fadcb6dc7f1415a3243", "testCaseId": "8a41af8eca098c4295d5e872d129839c", "fullName": "bms.testcases.test_02_usersync.TestUserSync#test_usersync_get_syncgroupid", "labels": [{"name": "epic", "value": "用户管理"}, {"name": "feature", "value": "用户同步管理-用户同步"}, {"name": "story", "value": "编辑，获取同步根组id接口"}, {"name": "parentSuite", "value": "bms.testcases"}, {"name": "suite", "value": "test_02_usersync"}, {"name": "subSuite", "value": "TestUserSync"}, {"name": "host", "value": "OFYI7JX9DURTDV6"}, {"name": "thread", "value": "9620-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "bms.testcases.test_02_usersync"}]}